<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinanceFormClass</class>
 <widget class="QDialog" name="FinanceFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FinanceForm</string>
  </property>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Поступление или расход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>240</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>240</y>
     <width>90</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>50</y>
     <width>641</width>
     <height>137</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="horizontalSpacing">
     <number>9</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="payerLayout">
      <item>
       <widget class="QLabel" name="payerLbl">
        <property name="text">
         <string>Плательщик</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="payerBox"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="responsibleLayout">
      <item>
       <widget class="QLabel" name="responsibleLbl">
        <property name="text">
         <string>Ответственный</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="responsibleBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QVBoxLayout" name="categoryLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="categoryLbl">
        <property name="text">
         <string>Категория</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="categoryBox">
        <item>
         <property name="text">
          <string>Нераспределенные</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Продажа товаров</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Оказание услуг</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Получение кредита</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Проценты к получению</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Прочий доход</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Перевод между счетами</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Нераспределенная сумма</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="recipientLayout">
      <item>
       <widget class="QLabel" name="recipientLbl">
        <property name="text">
         <string>Получатель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recipientBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="orgAccount">
      <item>
       <widget class="QLabel" name="orgAccountLbl">
        <property name="text">
         <string>Счет организации</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="orgAccountBox">
        <item>
         <property name="text">
          <string>Наличные</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Расчетный счет</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QVBoxLayout" name="communicationLayout">
      <item>
       <widget class="QLabel" name="communicationLbl">
        <property name="text">
         <string>Связь</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="communicationBox"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="dateLayout">
      <item>
       <widget class="QLabel" name="dateLbl">
        <property name="text">
         <string>Дата оплаты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="date"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="cashLayout">
      <item>
       <widget class="QLabel" name="cashLbl">
        <property name="text">
         <string>Сумма</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cash"/>
      </item>
     </layout>
    </item>
    <item row="2" column="2" colspan="2">
     <layout class="QVBoxLayout" name="noteLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="noteLbl">
        <property name="text">
         <string>Примечание</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="note"/>
      </item>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="stateLayout">
      <item>
       <widget class="QLabel" name="stateLbl">
        <property name="text">
         <string>Статус проведения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stateBox">
        <item>
         <property name="text">
          <string>ДДС и ПиУ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ДДС</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ПиУ</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Не проведен</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="warningLbl">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверьте корректность данных</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
